### PR TITLE
Section

### LINKED ISSUE #3: Create migration for Section
id:integer
title: string
icon: string

### LINKED ISSUE #6: Create many-to -many relationship for UserSections
UserSection
user_id
section_id

### LINKED ISSUE #11: Query for all Sections by user id
- [x] Can get all phrases through section

### PR DESCRIPTION
**Pull request by EricLarson2020**
### Description
Added Section model and query method
Added Usersections model and sections to user query method
Added testing for the above

### Type of change
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [yes] Refactor (non-breaking change which maintains existing functionality)
 - [yes] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: app/graphql/types/query_type.rb [hunk 1]
```diff
@@ -1,20 +1,26 @@
 module Types
   class QueryType < Types::BaseObject
-    field :user, Types::UserType, null: false do
```
Comment (EricLarson2020) [inline app/graphql/types/query_type.rb:3]: Made query for section, can now query sections on user

File: app/graphql/types/section_type.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+module Types
+  class SectionType < Types::BaseObject
```
Comment (EricLarson2020) [inline app/graphql/types/section_type.rb:2]: defines section types, can access users through sections

File: app/models/section.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class Section < ApplicationRecord
```
Comment (EricLarson2020) [inline app/models/section.rb:1]: Defined joins relationship

File: config/application.rb [hunk 4]
```diff
@@ -10,7 +10,10 @@
 require "action_mailer/railtie"
 require "action_view/railtie"
 require "action_cable/engine"
-# require "sprockets/railtie"
```
Comment (EricLarson2020) [inline config/application.rb:13]: Added this back in for heroku

File: app/graphql/types/query_type.rb [hunk 5]
```diff
@@ -1,20 +1,26 @@
 module Types
   class QueryType < Types::BaseObject
-    field :user, Types::UserType, null: false do 
+    field :sections, [Types::SectionType], null: false
+
+    def sections
+      Section.all
+    end
+
```
Comment (cgaddis36) [inline app/graphql/types/query_type.rb:8]: ahh I missed ActiveRecord

File: app/graphql/types/section_type.rb (hunk 2 above)
Comment (cgaddis36) [reply app/graphql/types/section_type.rb:2]: nice work here

File: config/application.rb (hunk 4 above)
Comment (cgaddis36) [reply config/application.rb:13]: I believe we want this commented out still? @sagemlee probably has more insight than I though

File: app/graphql/types/query_type.rb (hunk 5 above)
Comment (sagemlee) [reply app/graphql/types/query_type.rb:8]: Wish I could say the same

File: config/application.rb [hunk 6]
```diff
@@ -10,7 +10,10 @@
...
+
+require "sprockets/railtie"
```
Comment (sagemlee) [inline config/application.rb]: So this actually shouldnt be in here, the line was what was causing heroku to break. So it should be commented out.

**Review by sagemlee — CHANGES_REQUESTED** [review]

File: config/application.rb (hunk 6 above)
Comment (EricLarson2020) [reply config/application.rb]: Should be updated now

**Review by sagemlee — APPROVED** [review]

**Review by cgaddis36 — APPROVED** [review]

**Review by cgaddis36 — APPROVED** [review]

**Pull request merged**